### PR TITLE
[DYN-2870] Graph Workspaces Coverage

### PR DESCRIPTION
**Pull request by Astul-Betizagasti**
### Purpose

Increase code coverage in the Workspaces Folder

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

Aaron Tang (@QilongTang )
Michael Kirschner (@mjkkirschner )

### FYIs

Alfredo Pozo (@alfredo-pozo )

### CODE REVIEW

File: test/DynamoCoreTests/Graph/Workspaces/CustomNodeWorkspaceModelTests.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            Assert.IsNotNull(nodeWorkspace);
+
+            OpenTestFile(@"core\CustomNodes", "TestAdd.dyn");
+            var homeWorkspace = CurrentDynamoModel.CurrentWorkspace as HomeWorkspaceModel;
+            Assert.NotNull(homeWorkspace);
+
+            //Changes the Name property so that the event is raised
+            nodeWorkspace.Name = "NewNodeName";
+
+            var resultNode = homeWorkspace.Nodes.First(x => x.Name == "NewNodeName");
```
Comment (QilongTang) [inline test/DynamoCoreTests/Graph/Workspaces/CustomNodeWorkspaceModelTests.cs:35]: `FirstOrDefault`?

File: test/DynamoCoreTests/Graph/Workspaces/CustomNodeWorkspaceModelTests.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            Assert.IsNotNull(resultNode);
+            Assert.IsTrue(resultNode.Name == "NewNodeName");
```
Comment (QilongTang) [inline test/DynamoCoreTests/Graph/Workspaces/CustomNodeWorkspaceModelTests.cs:37]: We can probably remove this line as it is duplicate with the name search above

File: test/DynamoCoreTests/Graph/Workspaces/HomeWorkspaceModelTests.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        {
+            //Loads a file with a valid graph on the Home workspace
+            OpenTestFile(@"core\number", "TestBigNumber.dyn");
+            var homeWorkspace = CurrentDynamoModel.CurrentWorkspace as HomeWorkspaceModel;
+            Assert.NotNull(homeWorkspace);
+
+            //Gets results from raised event
+            bool eventWasRaised = false;
+            string objectFileName = "";
+            homeWorkspace.SetNodeDeltaState += delegate(object workspace, DeltaComputeStateEventArgs d)
```
Comment (QilongTang) [inline test/DynamoCoreTests/Graph/Workspaces/HomeWorkspaceModelTests.cs:32]: When do we unsubscribe?

Comment (Astul-Betizagasti) [reply test/DynamoCoreTests/Graph/Workspaces/HomeWorkspaceModelTests.cs:32]: At the moment we don't, I'll look into it and update the pr

**Pull request merged**